src/v3/src/components/Widget/style.module.css

```diff
@@ -0,0 +1,237 @@
+  }
+}
+@media only screen and (max-height: 750px) {
+  :global(.okta-container .applogin-banner .applogin-container) {
+    padding-block: 10px;
+    padding-inline: 0;
+  }
+}
+@media only screen and (max-height: 660px) {
+  :global(.okta-container .applogin-banner .applogin-container) {
+    padding-block: 5px;
+    padding-inline: 0;
+  }
+}
+:global(.okta-container .applogin-banner .applogin-container h1) {
+  font-size: 24px;
+  font-weight: lighter;
+  line-height: 26px;
+}
+@media only screen and (max-height: 660px) {
+  :global(.okta-container .applogin-banner .applogin-container p) {
+    display: none;
+  }
+}
+:global(.okta-container .applogin-banner .applogin-app-logo) {
+  display: inline-block;
+  vertical-align: middle;
+  margin-inline-start: 5px;
+}
+:global(.okta-container .applogin-banner .applogin-app-logo img) {
+  max-block-size: 26px;
+}
+@media only screen and (max-width: 600px) {
+  :global(.login-bg-image) {
+    background-image: none !important;
+    background-color: #fff !important;
+    -webkit-filter: unset !important;
+    filter: unset !important;
+  }
+}
+/*! CSS Used from: https://op3static.oktacdn.com/assets/loginpage/css/loginpage-theme.c1227d73b70be13e51aae80fe238b0ae.css */
+a,
+body,
+div,
+form,
+h1,
+h2,
+html,
+iframe,
+img,
+p {
+  background: transparent;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  margin: 0;
+  outline: 0;
+  padding: 0;
+  vertical-align: baseline;
+}
+body {
+  line-height: 1;
+}
+:focus {
+  outline: 0;
+}
+input[type="hidden"] {
+  display: none !important;
+}
+html {
+  block-size: 100%;
+}
+body {
+  background-color: #f9f9f9;
+  color: #777;
+  font-family: proxima nova, Arial, Helvetica, sans-serif;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 1.4;
+  min-block-size: 100%;
+  min-inline-size: 100%;
+  overflow: auto;
+  position: relative;
+}
+:global(#inactive-tab-main-div .inactive-tab-container) {
+  background: #fff;
+  margin-block: 0;
+  margin-inline: auto;
+  padding-block: 30px 75px;
+  padding-inline: 90px;
+  text-align: center;
+  inline-size: 32rem;
+}
+:global(#inactive-tab-main-div .inactive-tab-container .inactive-tab-header) {
+  padding-block-end: 30px;
+}
+:global(#inactive-tab-main-div .inactive-tab-container .inactive-tab-header img) {
+  max-block-size: 50px;
+  max-inline-size: 100px;
+}
+:global(#inactive-tab-main-div .inactive-tab-container .inactive-tab-content-title) {
+  color: #5e5e5e;
+  font-weight: 600;
+  padding-block-end: 30px;
+}
+@media only screen and (max-device-width: 600px) {
+  :global(#inactive-tab-main-div .inactive-tab-container) {
+    margin: auto;
+    inline-size: 100%;
+  }
+}
+:global(.hide) {
+  display: none;
+}
+/*! CSS Used from: https://oie-4695462.oktapreview.com/api/internal/brand/theme/style-sheet?touch-point=SIGN_IN_PAGE&v=f991ebb043f05a9393f38ace2d7dafefc25baa57484ea41b8969a14da06b2d45c27314de737b7014d6aad7d515925099 */
+:global(.tb--background) {
+  background-color: #f9f9f9 !important;
+}
```

Comment:
Correct me if I am wrong but I believe this can be removed right? This content comes from the loginpagetheme.css which is actually loaded by the loginpage_backbone.jsp so no need to define them here as well. Additionally, when I test this using the embedded scenario, I can see a bunch of styles applied that should not be applied. 

cc: @lesterchoi-okta

Reply:
Nevermind, I am extremely confident this can be removed so I removed it.